Kickstarter-iOS/Features/BlockingUsers/BlockUserActionSheetView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct BlockUserActionSheetView: View {
+  var blockUser: () -> Void
+  var viewProfile: (() -> Void)?
+
+  var body: some View {
+    // Scott TODO: Use localized strings once translations are done
+    EmptyView()
+      .confirmationDialog("", isPresented: .constant(true), titleVisibility: .hidden) {
+        if let viewProfile = viewProfile {
+          Button("View profile") {
+            viewProfile()
+          }
+        }
+
+        Button(role: .destructive) {
+          blockUser()
+        } label: {
+          Text("Block this user")
+        }
+      }
+  }
+}
```

Comment:
Question: Do we need a custom action sheet here, or could we just use a `UIActionSheet`? That'd cut down the amount of code you need pretty substantially.

Reply:
We don't need it to be custom, no. I thought it would be more fun to build in SwiftUI here. It's a very simple view and I don't think it adds overhead. 
We definitely could present an action sheet using UIAlertController instead. I'm happy to re-work it if you think there's a strong argument. I can see one for maintaining existing patterns, but is there a performance gain here?